File: primes.cpp
#include <iostream>
#include <chrono>

using namespace std;

int main()
{
    auto start = chrono::high_resolution_clock::now();
    cout << "Started in C++" << endl;
    int i = 0;
    int primes = 0;
    while (i < 150000)
    {
        bool isPrime = true;
        for (int j = 2; j < i; j++)
        {
            if (i % j == 0)
            {
                isPrime = false;
                break;
            }
        }
        if (isPrime)
        {
            primes++;
        }
        i++;
    }
    auto end = chrono::high_resolution_clock::now();
    auto time = chrono::duration_cast<chrono::milliseconds>(end - start).count();
    cout << "Execution time: " << time / 1000.0 << " seconds" << endl;
    cout << "Primes found: " << primes << endl;
    cout << "Primes per second: " << (primes / (time / 1000.0)) << endl;

    cin.get();
    return 0;
}

File: primes.py
import time
start = time.time()
print("Started in Python")
i = 1
primes = 0
while i < 150000:
    i += 1
    isPrime = True
    for j in range(2, i):
        if i % j == 0:
            isPrime = False
            break
    if isPrime:
        primes += 1
end = time.time()
executionTime = end - start
print("Execution time: " + str(executionTime))
print("Primes found: " + str(primes))
print("Primes per second: " + str(primes / executionTime))

input("Press something to exit...")
